StyleSheet.css

```diff
@@ -563,3 +564,168 @@ li {
 .box img:hover {
   transform: scale(108%);
 }
+
+.modal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  padding-top: 100px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+.modal-content {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  position: absolute;
+  width: 343px;
+  height: 713px;
+  left: calc(50% - 343px / 2);
+  top: 60px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.closeModal {
+  left: 25%;
+  right: 25%;
+  top: 25%;
+  bottom: 25%;
+  font-size: 30px;
+  border: 2px #67798e;
+}
+
+.closeModal:hover,
+.closeModal:focus {
+  color: red;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.title-close-in-row {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  width: 311px;
+  height: 40px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.primary-text-modal {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  gap: 12px;
+  width: 311px;
+  height: 84px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.snapshotModal {
+  border-radius: 8px;
+  margin: auto auto auto auto;
+}
+
+.button-in-row {
+  width: 311px;
+  height: 56px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin-top: 63px;
+  margin-bottom: 16px;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px 14px;
+}
```

Comment:
- [ ] As requested by the previous reviewer https://github.com/AlexisJW/set-up-myFirst-mobile/pull/15#discussion_r1054532528 please align the buttons to match the figma template, this will improve the design of our modals, you can use below code to achieve this
```
.button-in-row {
  justify-content: flex-end;
}
```
you can also align the social links in the popup using 
```
.right-block {
  align-items: center;
}
```
currently it is looking like this :point_down: 
![image](https://user-images.githubusercontent.com/97521557/208967822-7523993d-670e-422f-97f7-461a12a42597.png)